nephio-5gc-controller/apis/nf/v1alpha1/nf_types.go

```diff
@@ -17,6 +17,6 @@ limitations under the License.
 package v1alpha1
 
 type Endpoint struct {
-	Ipv4Addr []string `json:"ipv4Addr"`
-	Gwv4Addr *string  `json:"gwv4addr,omitempty"`
+	IP        []string `json:"ip"`
```

Comment:
@s3wong are these IPs or CIDRs or is either OK? We may want to use a different name if a CIDR may be in here rather than a specific IP.

Reply:
It is meant for CIDR input...

Reply:
Isn't this same type used in `UPFDeployment` where they are IPs? Or are they still CIDRs there? If they are CIDRs there, where do the actual IPs get stored?

Reply:
So is Endpoint more like a Subnet then, rather than an individual IP configuration?

Reply:
I'm going to merge it like this, and we'll correct it as we integrate with @henderiw's IPAM if needed.